src/aiida_quantumespresso/parsers/base.py

```diff
@@ -25,39 +88,127 @@ def emit_logs(self, logging_dictionaries, ignore=None):
                 'error': ['Self-consistency was not achieved']
             }
 
-        :param logging_dictionaries: log dictionaries
+        :param logs: log dictionaries
         :param ignore: list of log messages to ignore
         """
         ignore = ignore or []
 
-        if not isinstance(logging_dictionaries, (list, tuple)):
-            logging_dictionaries = [logging_dictionaries]
+        if not isinstance(logs, (list, tuple)):
+            logs = [logs]
 
-        for logs in logging_dictionaries:
+        for logs in logs:
             for level, messages in logs.items():
                 for message in messages:
 
-                    if message is None:
-                        continue
-
                     stripped = message.strip()
 
-                    if not stripped or stripped in ignore:
```

Comment:
Same as https://github.com/aiidateam/aiida-quantumespresso/pull/756/files#r1192076537. If we have an empty log message something must have gone wrong that we should fix?

Reply:
I think this may have accommodated `None` versions as well, but you are probably right that this would be a problem in our code, and so we rather fix it then rely on this.